### ISSUE TITLE
pongoOS device not found

### ISSUE THREAD

**Issue by haen7** [checkra1n/BugTracker#1009]
**Tell us about your setup:**
1. What iDevice are you using? iPhone 6
2. On what version of iOS is it? 12.4
3. What version of checkra1n are you using? GUI or CLI? 0.9.8.1 CLI (linux and macOS)
4. What is your host system (OS version? Hackintosh? VM? etc.)? Ubuntu 18.04, macOS 10.15.4
5. How are you connecting to the device (USB-A? USB-C? Apple/3rd party cable? Through a USB hub?)? 3rd party USB-A/Anker power line II

**What are the steps to reproduce the issue?**
1. boot iPhone using checkra1n -cvp 

**What do you expect, and what is happening instead?**

 Device boots into pongoOS 1.0.2-67136f43 (EL3) state but Ubuntu/macOS can not detect pongoOS USB.
 `lsusb` command does not show iPhone pongoOS in Ubuntu.
 `system_profiler SPUSBDataType` command does not show iPhone pongoOS in macOS.

 It works fine on iPhone 6s but issue occurs on iPhone 6. 

**Does the issue also occur if you enable Safe Mode?**

**Any other info, error logs, screenshots, ...?**
Here is the output when trying to run command in pongoOS using `issue_cmd.py`:
```
~/Desktop/sources/pongoOS/scripts$ sudo python3 issue_cmd.py help
Traceback (most recent call last):
  File "issue_cmd.py", line 27, in <module>
    raise ValueError('Device not found')
ValueError: Device not found 
```

**Comment by woachk** [comment]
https://github.com/checkra1n/pongoOS/issues/6

Same issue as that one.

We do not currently support A7 and A8 for pongoOS USB... or rather it's broken.

**Comment by ForceBru** [comment]
@woachk, it looks different: my iPhone 6+ _is_ recognised properly, and this line doesn't return `None`:

https://github.com/checkra1n/pongoOS/blob/c5c8261a7588641b981d076689d07f1d06360421/scripts/issue_cmd.py#L25

But I'm not booting into pongoOS, actually, I'm just in DFU mode. Should I be booting into pongoOS with Checkra1n first?

**Comment by woachk** [comment]
Hello,

You can't send commands to pongoOS before pongoOS starts up of course. :-)

**Comment by ForceBru** [comment]
@woachk, so the `0x21` in `dev.ctrl_transfer(0x21, 4, 0, 0, 0)` is pongoOS-specific, right?

**Comment by woachk** [comment]
@ForceBru yes

**Comment by haen7** [comment]
This bug is fixed in the latest version of pongoOS (2.4.0).
